src/shared/components/error-screen/url-errors/UrlError.scss

```diff
@@ -0,0 +1,37 @@
+@import 'src/styles/common';
+
+.topBar {
+  height: 40px;
+  background-color: $light-grey;
+  box-shadow: 0 3px 5px $global-box-shadow;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 88px;
+}
+
+.alertButton {
+  --alert-icon-diameter: 30px;
+  font-size: 0;
+}
+
+.errorMessage {
+  color: $red;
+  font-weight: $bold;
+  margin-top: 34px;
+}
+
+.suggestion {
+  margin-top: 15px;
+}
+
+.continueButton {
+  --button-link-color: #{$green};
+}
+
+.continueButtonWrapperExtraPadded {
```

Comment:
Why do we need to include `ExtraPadded`?

Reply:
Andrea had two designs: [one](https://xd.adobe.com/view/b6d9a69d-468f-44b1-9333-419e228aa5ff-e38a/screen/3a909686-ce1c-4f93-b893-671d366435cf?fullscreen) for genome browser and entity viewer:

![xd adobe com_view_b6d9a69d-468f-44b1-9333-419e228aa5ff-e38a_screen_3a909686-ce1c-4f93-b893-671d366435cf](https://user-images.githubusercontent.com/6834224/180330255-adf92506-cb00-410a-81e9-18d480f24ceb.png)

and [another](https://xd.adobe.com/view/b6d9a69d-468f-44b1-9333-419e228aa5ff-e38a/screen/a866f4ab-afc2-4322-ab08-78e2c6ce6759?fullscreen) for the species page:

![Uploading xd.adobe.com_view_b6d9a69d-468f-44b1-9333-419e228aa5ff-e38a_screen_3a909686-ce1c-4f93-b893-671d366435cf (1).png…]()

I am not quite sure how to pass this information into the error screen cleanly.

Reply:
I had a chat with Andrea, and we ended up tweaking several classes:

<img width="819" alt="image" src="https://user-images.githubusercontent.com/6834224/180452346-b2d2d173-1f5f-457e-ba46-06993d211bbe.png">

I renamed the extra padded class as well.